lib/y2snapper_common.pm

```diff
@@ -11,6 +11,43 @@ use testapi;
 use utils;
 use version_utils;
 
+=head2 y2snapper_select_current_conf
+
+ y2snapper_select_current_conf($ncurses);
+
+Select Current Configuration on Snapshots screen
+C<$ncurses> is used to check if it is ncurses.
+
+=cut
+sub y2snapper_select_current_conf {
+    my ($self, $ncurses) = @_;
+    $ncurses //= 0;
+    if ($ncurses) {
+        send_key 'alt-tab';    # Focus Current Configuration selection box
+        send_key 'down';       # Expand test configuration selection box
+        send_key 'down';       # Select test configuration
```

Comment:
also consider send_key_until_needlematch. looks better practice than send_key twice here

Reply:
There are two reason to not do it, one is that this test is running in many other products and we should be very conservative creating needle and the second is the one in the previous comment, that focused combobox depending on the product is done with different font, so the needle does not help in some products, as there is no much different between selection and not focused control.